src/views/admin/PersonCreationView.vue

```diff
@@ -199,29 +269,61 @@
   }
 
   async function createPerson(): Promise<void> {
+    // Function to format a date in dd.MM.yyyy format to ISO 8601
+    function formatDateToISO(date: string | undefined): string | undefined {
+      if (date) {
+        // Split the date by '.' to extract day, month, and year
+        // eslint-disable-next-line @typescript-eslint/typedef
+        const [day, month, year] = date.split('.').map(Number);
+
+        if (day && month && year) {
+          // Create a new Date object with the extracted parts
+          // Alwyays adding 1 day to the date because for example if the Befristung is chosen as 20.05.2024 then it should be guilty until 20.05.2024 23:59
```

Comment:
```suggestion
          // Alwyays adding 1 day to the date because for example if the Befristung is chosen as 20.05.2024 then it should be valid until 20.05.2024 23:59
```